spec/03-plugins/15-response-transformer/02-body_transformer_spec.lua

```diff
@@ -135,6 +135,12 @@ describe("Plugin: response-transformer", function()
         assert.same({a = {}}, body_json)
         assert.equals('[]', cjson.encode(body_json.a))
       end)
+
+      it("plain text", function()
+        local body = "test"
+        local json_body = body_transformer.transform_json_body(conf, body)
+        assert(json_body == nil)
```

Comment:
```suggestion
        local json_body, err = body_transformer.transform_json_body(conf, body)
        assert.is_nil(json_body)
        assert.equals(err, "failed parsing json body")
```
the body was returned `nil` even before this fix, so this is not testing the change right now. I would suggest checking the expected error message here.